books/free-programming-books-bn.md

```diff
@@ -53,6 +60,11 @@
 * [ডেভসংকেত: বাংলা চিটশিটের ভান্ডার](https://devsonket.com) - Devsonket Team
 
 
+### PHP
+
+* [পিএইচপি শিখুন বাংলায় - মিজানুর রহমান ](https://www.amarbooks.org/download.php?id=17935#pdf) - Miznaur Rahman(PDF, HTML)
```

Comment:
Is this a legitimate website?

Reply:
Yes. This website is legitimate. You can download the book/read the book online after entering the captcha, and around 22k people have viewed/downloaded it from here. Thank you